sql/ts/tests/apitests.ts

```diff
@@ -487,14 +487,21 @@ async function testJSPrivacy(skdb: SKDB, skdb2: SKDB) {
 
   await skdb.exec("INSERT INTO test_pk VALUES (37, 42, 'my_group');");
 
-  let user1_view = await skdb.exec(
-    "SELECT * FROM test_pk WHERE skdb_access='my_group';",
-  );
-  let user2_view = await skdb2.exec(
-    "SELECT * FROM test_pk WHERE skdb_access='my_group';",
-  );
-  expect(user1_view).toHaveLength(1);
-  expect(user2_view).toHaveLength(0);
+  expect(
+    await waitSynch(
+      skdb,
+      "SELECT * FROM test_pk WHERE skdb_access='my_group';",
+      (rows) => rows.length == 1,
+    ),
+  ).toHaveLength(1);
```

Comment:
waitSynch runs the query on the remote side, but it looks like here - and all your other cases - were testing that things are correct on the client-side. Is this what you intended?

Reply:
And I've just noticed the server flag defaulting to false. Ignore this comment.